docs/FAQ.md

```diff
@@ -0,0 +1,224 @@
+# FAQ
+
+## _Definitions and concepts_
+
+### What is a notebook?
+
+A "computational notebook" is a type of program that allows mixing formatted text and executable code to create documents with runnable examples. Notebooks are an example of [literate programming](https://en.wikipedia.org/wiki/Literate_programming). A notebook has "cells" which are different text regions. There are commonly three different kinds of cells: 
+
+* _Code cells_ contain runnable code.
+* _Output cells_ contain the result from the last execution of the associated code cell. 
+* _Markdown cells_ are display-only and can be used to edit and display richly-formatted (but static) text, including hyperlinks, images, diagrams, and so on. 
+
+Project Jupyter, which is probably the most well-known notebook technology, has grown from its origins in scientific and academic environments to become a mainstay in industry for data analysis and data science. Other use cases include interactive documentation and learning materials, troubleshooting guides, and self-guiding automation scripts that can capture structured or visual log output.
+
+### What is Jupyter Notebook?
+
+Jupyter Notebook is a browser-based notebook UI (or "frontend") from Project Jupyter that can edit and run notebooks using a Jupyter kernel and the `.ipynb` file format.
+
+### What is JupyterLab?
+
+JupyterLab is another browser-based frontend for Jupyter kernels, similar to Jupyter Notebook, but richer and more extensibible.
+
+### What is .NET Interactive?
+
+.NET Interactive is a [.NET tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools) ([`dotnet-interactive`](https://www.nuget.org/packages/Microsoft.dotnet-interactive)) containing a general-purpose engine for interactive programming including, but not limited to, use with notebooks. It includes suport for a number of languages, including C#, F#, PowerShell, and JavaScript, as well as the ability to load additional languages such as SQL and Python.
+
+### What is the difference between .NET Interactive and Jupyter?
+
+The .NET Interactive kernel (i.e. the `dotnet-interactive` [.NET tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools)) is a Jupyter-compliant kernel that can be used with Project Jupyter just like any other kernel. Like other Jupyter kernels, .NET Interactive isn't opinionated about which frontend you use. 
+
+The Polyglot Notebooks extension for VS Code isn't required to use the .NET Interactive kernel, but it does provide access to some additional functionality that isn't typical of Jupyter frontends, such as the ability to switch languages (i.e. subkernels) on a per-cell basis.
```

Comment:
It would be great if we could add a link for the polyglot notebooks reference to the section below.